src/main/java/kg/jarkyn/server/Incoming/ParsedRequest.java

```diff
@@ -0,0 +1,19 @@
+package kg.jarkyn.server.Incoming;
```

Comment:
oh really, i didn't know, I saw that java uses them here and there, so went with it